src/front/DataInspector.js

```diff
@@ -2,7 +2,18 @@ L.TileLayer.XRay = L.TileLayer.extend({
 
     getTileUrl: function (tilePoint) {
         this.options.version = Date.now();
-        this.options.showLayer = L.K.Config.dataInspectorLayer || '';
+        var keys = Object.keys(L.K.Config);
+        if (L.K.Config['dataInspectorLayer___all__'] === true) {
```

Comment:
I thought so, too. Because it's in a separate class, it's difficult to get variables out of the dataInspector object. Can you explain to me how to get L.K.Config.dataInspector to become a dictionary with multiple layers? I don't know enough about the framework - I tried to give the checkboxes ids like "dataInspector[layerName]".